aws/policy/application-services.yaml

```diff
@@ -163,6 +163,12 @@ Statement:
       - logs:PutMetricFilter
     Resource:
       - 'arn:aws:logs:{{ aws_region }}:{{ aws_account_id }}:log-group:ansible-test*'
+  - Sid: DescribeEventRules
+    Effect: Allow
+    Action:
+      - events:DescribeRule
```

Comment:
I guess we need also `CreateRule` and `DeleteRule`. Zuul throws only the first AccessDenied error with missing DescribeRule and stops, because it is necessary for the first request.

Reply:
Adjusted accordingly